README.rst

```diff
@@ -21,9 +21,9 @@ Full documentation on `read the docs`_.
 Requirements
 ------------
 
-* **Python**: 3.4, 3.5, 3.6, 3.7
-* **Django**: 1.11, 2.0, 2.1, 2.2
-* **DRF**: 3.8+
+* **Python**: 3.5, 3.6, 3.7, 3.8
+* **Django**: 1.11, 2.0, 2.1, 2.2, 3.0
+* **DRF**: 3.10+
```

Comment:
Not sure if this is true, but tests are only running against `djangorestframework==3.10.*`.